testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -57,3 +57,21 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I click the Add Product button
     And I wait until I see "SUSE Linux Enterprise Server 15 x86_64" product has been added
     Then the SLE15 products should be added
+
+@long_test
```

Comment:
What makes the difference in this scenario to need the special tag of long_test, comparing with the other scenarios?
I would say we don't need it here.

Reply:
it's just that we don't need to fully sync that product if we don't run the long tests

Reply:
Well, it will also mean that we will not run this scenario if we don't run the long tests, is that what we want? This product will be only necessary when running long tests? Otherwise, I would remove the tag from that one.

Reply:
Ok, as we only need at the same scenarios that we also have the tag, let's keep it here for now.

Reply:
As of today it's not needed anywhere else.